infra/modules/application-resources.bicep

```diff
@@ -321,6 +311,8 @@ module sqlDatabase '../core/database/sql-database.bicep' = {
     } : null
     sku: deploymentSettings.isProduction ? 'Premium' : 'Standard'
     zoneRedundant: deploymentSettings.isProduction
+    users: deploymentSettings.principalId != null ? [ deploymentSettings.principalId ] : []
```

Comment:
this appears to give the local developer db_owner. that explains the changes to the developer-experience.md file.

Does the .NET code still have permission to run EF migrations on startup? the .NET code runs as user-assigned managed identity to connect, are there roles assigned for that?

Reply:
> Does the .NET code still have permission to run EF migrations on startup?

Yes

>  the .NET code runs as user-assigned managed identity to connect, are there roles assigned for that?

that user-assigned managed identity is the admin on the sql server as it already was. The only real change here is that the developer now connects with managed identity - the app already did